.github/workflows/run-k8s-tests-on-sev.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI | Run kubernetes tests on SEV
```

Comment:
This workflow is almost identical to run-k8s-tests-on-tdx.yaml, except for the vmm, runner node and perhaps the kubeconfig (I explain later).

So one possibility is to combine the TDX and SEV (possible others TEEs on future) into a single workflow. Something like:
 * Rename to run-k8s-tests-on-tdx.yaml -> run-k8s-tests-on-tee.yaml
 * Change the workflow:
```
diff --git a/.github/workflows/run-k8s-tests-on-tee.yaml b/.github/workflows/run-k8s-tests-on-tee.yaml
index 1777a16c8..d497a6667 100644
--- a/.github/workflows/run-k8s-tests-on-tee.yaml
+++ b/.github/workflows/run-k8s-tests-on-tee.yaml
@@ -1,4 +1,4 @@
-name: CI | Run kubernetes tests on TDX
+name: CI | Run kubernetes tests on TEE
 on:
   workflow_call:
     inputs:
@@ -19,9 +19,17 @@ jobs:
       matrix:
         vmm:
           - qemu-tdx
-    runs-on: tdx
+          - qemu-sev
+        include:
+          - vmm: qemu-tdx
+            runner: tdx
+            kubeconfig: /etc/rancher/k3s/k3s.yaml
+          - vmm: qemu-sev
+            runner: sev 
+            kubeconfig: /path/to/kubeconfig
+    runs-on: ${{ matrix.runner }}
     env:
-      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+      KUBECONFIG: ${{ matrix.kubeconfig }}
     steps:
```

Reply:
This could work. My only concern is that sev and snp will need to add additional bats files to the K8S_TEST_UNION, but we can accommodate in the tee yaml.

Reply:
I talked with @ryansavino and we decided to not merge the worflows for now. However, I will be opening an issue for the consolidation, that we work to do when the pipelines are all merged and stable.

Reply:
As discussed a few times in the past, we're hoping that the SEV / SNP additional bats tests will be generic enough to accommodate other TEEs as well (with internal implementations being different), thus I do believe it does make sense to merge the files.

Reply:
Yes, refactoring the sev/snp tests will be a requirement. But IIUC at this point we are running the generic k8s tests, none related with TEE. Also @ryansavino has started the work with refactoring the sev tests.